Resources/Prototypes/Datasets/tips.yml

```diff
@@ -32,30 +28,99 @@
   - "The AME is a high-priority target and is easily sabotaged. Make sure to set up the Singularity or Solars so that you don't run out of power if it blows."
   - "You can add labels to any item, including food or pill canisters, using a hand labeler."
   - "Riot armor is significantly more powerful against opponents that aren't using guns compared to regular armor."
-  - "As a Security Officer, remember that correlation does not equal causation. Someone may have just been at the wrong place at the wrong time!"
   - "As a ghost, you can use the Verb Menu to orbit around and follow any entity in game automatically."
+  - "As a Mouse, you can alt-click food to eat it."
   - "As a Traitor, you may sometimes be assigned to hunt other traitors, and in turn be hunted by others."
   - "As a Traitor, the syndicate headset can be used to communicate with any other traitors who may have purchased it through a secure channel."
+  - "As a Traitor, never forget the power of the syndicate toolbox. For only 2 telecrystals, you can get a bunch of handy tools to boost you in an emergency, along with the toolbox itself packing a decent punch."
+  - "As a Traitor, never underestimate the web vest. It may not provide space protection, but its cheap cost and robust protection makes it handy for trigger-happy foes."
   - "As a Traitor, any purchased grenade penguins won't attack you, and will explode if killed."
+  - "As a Traitor, remember that power sinks create a loud sound and alert the crew after running for long enough. Try to hide them in a tricky-to-find spot, or reinforce the area around them so that they're harder to reach."
+  - "As a Traitor, be mindful of the power of destroying telecomms. It'll be a lot harder for people to call you out if they can't talk to others over long distances!"
+  - "As a Traitor, plasma gas is an an excellent way to create chaos. It can be ignited to make an area extra-uninhabitable, and can cause toxin damage to those that inhale it."
   - "As a Nuclear Operative, stick together! While your equipment is robust, your fellow operatives are much better at saving your life: they can drag you away from danger while stunned and provide cover fire."
   - "As a Nuclear Operative, communication is key! Use your radio to speak to your fellow operatives and coordinate an attack plan."
+  - "As a Nuclear Operative, remember that stealth is an option. It'll be hard for the captain to fight back if he gets caught off-gaurd by what he thinks is just a regular passenger!"
   - "You can examine your headset to see which radio channels you have available and how to speak in them."
   - "As a Salvage Technician, always carry a GPS on you and take note of the station's coordinates in case your salvage is lost to space."
+  - "As a Salvage Technician, you can use your proto-kinetic accelerator to move yourself in space when in a pinch."
```

Comment:
I mean, it does work, but it's a terrible way of moving, since the nerf.

Reply:
fixed? probably